src/ventus/wm/window.js

```diff
@@ -486,8 +489,23 @@ function(Emitter, Promise, View, WindowTemplate) {
 		},
 
 		move: function(x, y) {
-			this.x = x;
-			this.y = y;
+			var maxX = window.innerWidth - this._margin.right;
+			var maxY = window.innerHeight - this._margin.bottom;
```

Comment:
I agree with @amatiasq as well. Ventus should limit itself to the container, in that case this pull request is null and void. Thank you both! I will keep trying to help however next time I will think through the issue at hand a little more.